nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,196 @@
+from typing import Mapping, Union
+
+from nextcord.ext.commands import Cog, Command, Group, HelpCommand
```

Comment:
I think the base class pyright issue can be solved by using `from nextcord.ext import commands` and `commands.HelpCommand`, `commands.Cog`, `commands.Command`, etc. in the other locations
```suggestion
from nextcord.ext import commands
```